### ISSUE TITLE
OnClick Method (ChartConfig.Options.OnClick)

### ISSUE THREAD

**Issue by manon-ht** [ChristopherMWood/XamarinChartJSPlugin#2]
Hello !

Firstly, thank you verry much for your library. It is really usefull.

I am trying to find out a way to redirect to the detail page of my item when a user click on the label.
A metod seems to exist in the chartJs javascript library, which is the "onClick" event.
I can't find it in the "ChartConfig" class. Could you please tell me if you have implemented it ?
I guess you are working with MAUI now and your library is no longer maintain ?

Thank you, have a nice day !

**Comment by stretchycritter** [comment]
@manon-ht Thanks for your feedback! I'm glad you're finding it useful.

Unfortunately, I have not implemented the onClick event. I considered it when I first published the package but decided to leave it for if people requested it or if I ended up needing it. My plan, if I have time is to port it to .NET Maui, and rewrite it to have some of these features since I would just need to expose what ChartJS already does. If it's easy enough, I can port it back but no guarantees on when/if that will happen.
